content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -1091,7 +1091,7 @@ WindowsHostProcessContainers=true|false (BETA - default=true)<br/>
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">Comma-separated list of cipher suites for the server. If omitted, the default Go cipher suites will be used.<br/>
-Preferred values:
+Preferred values:<br/>
```

Comment:
Why this change?

Reply:
Reference [Insecure values](https://github.com/kubernetes/website/commit/01e0a9ebd16fb7650df82ae9253f19f25e0caa95#diff-a7a37ad9496edf0e4b1ef0cf4270571e43d83d29a9b8fd9c14b6a155a6746e38R1096)

Reply:
I checked against the [upstream code](https://github.com/kubernetes/kubernetes/blob/release-1.24/cmd/kubelet/app/options/options.go#L412-L413), no newline after `Preferred values:` and `Insecure values:`. I have deleted the `<br/>` after `Preferred values:` and `Insecure values:`. PTAL.